bundle/ios/GutenbergNativeTranslations.swift

```diff
@@ -27,39 +27,26 @@ private func dummy() {
     _ = NSLocalizedString("50 / 50", comment: "")
     _ = NSLocalizedString("70 / 30", comment: "")
     _ = NSLocalizedString("9:16", comment: "")
-    _ = NSLocalizedString("A calendar of your site’s posts.", comment: "")
```

Comment:
Do you know if these removals are of consequence? I wonder if they are accurate removals or potentially erroneous changes occurring due to outdated i18n files. Possibly related to https://github.com/wordpress-mobile/gutenberg-mobile/pull/3423? WDYT?

Reply:
I think we should only merge the Gutenberg reference update and not the rest, I'm guessing they were only present to be able to test the builds right @antonis? If so we can just update this PR to only have the GB commit hash change before merging.

Reply:
Thank you for reviewing this @dcalhoun! I really appreciate the time you put on this 🙇 

> Do you know if these removals are of consequence? I wonder if they are accurate removals or potentially

I'm not sure why the changes were introduced. It should be related with #3423.

> I think we should only merge the Gutenberg reference update and not the rest

I would agree with @geriux on this. I would just update the reference. The rest was just needed to be able to produce the App builds for testing.